ScnSoft-Education/LeetCodeTraining.Tests/PalindromeNumberTests/PalindromeNumberSolutionTests.cs

```diff
@@ -0,0 +1,93 @@
+﻿using LeetCodeTraining.PalindromeNumber;
+
+namespace LeetCodeTraining.Tests.PalindromeNumber
+{
+    public class PalindromeNumberSolutionTests
+    {
+        [Fact]
+        public void IsPalindrome_Negative()
+        {
+            //Arrange
+            var number = -123;
+            bool expectedResult = false;
+
+            //Act
+            var result = PalindromeNumberSolution.IsPalindrome(number);
+
+            //Assert
+            Assert.Equal(result, expectedResult);         
+        }
+
+        [Fact]
+        public void IsPalindrome_Zero()
+        {
+            //Arrange
+            var number = 0;
+            bool expectedResult = false;
```

Comment:
Expected result should be true.
https://en.wikipedia.org/wiki/Palindromic_number#:~:text=The%20first%2030%20palindromic%20numbers,the%20realm%20of%20recreational%20mathematics.